src/Octoshift/GithubApi.cs

```diff
@@ -143,6 +143,23 @@ public virtual async Task<string> GetOrganizationId(string org)
             return (string)data["data"]["organization"]["id"];
         }
 
+        public virtual async Task<string> GetEnterpriseId(string enterpriseName)
+        {
+            var url = $"{_apiUrl}/graphql";
+
+            var payload = new
+            {
+                // TODO: this is super ugly, need to find a graphql library to make this code nicer
```

Comment:
```suggestion
```

Reply:
I think you copied this comment from somewhere else in the code. I think at this point we've accepted that the GQL calls are as good as they're gonna get for now.